HelpSource/Reference/randomSeed.schelp

```diff
@@ -32,8 +32,11 @@ Array.fill(7, r);
 
 subsection::Inheriting Seeds
 
-Also it is possible to set the seed of the running thread that
-all threads started within will inherit.
+When a routine is created, it "inherits" its random number generator from the parent thread (link::Classes/Thread#.thisThread#thisThread::). If no particular randSeed is set (the normal case), it will tap the same stream of random number generators used by the parent.
```

Comment:
scare quotes unnecessary; capitalize Routine

Reply:
Also, this is still unclear to me -- are the two routines sharing the same *seed*, or the same *RNG object*? i.e. if r1 spawns r2 and then each calls `rand()`, will they receive the same number?